aws-greengrass-testing-platform/aws-greengrass-testing-platform-api/src/main/java/com/aws/greengrass/testing/platform/linux/LinuxNetworkUtils.java

```diff
@@ -51,6 +52,31 @@ public void recoverMqtt() throws InterruptedException, IOException {
         modifyMqttConnection(DISABLE_OPTION);
     }
 
+    @Override
```

Comment:
Can simplify to move common code to add_or_delete method. Delete differs from add only in one parameter to the command.

Reply:
I think we can extend the method in the future and according **Single Responsibility** it's wrong.